testsuite/tests/memory-model/publish.ml

```diff
@@ -1,7 +1,8 @@
 (* TEST
   modules="opt.ml barrier.ml hist.ml shared.ml run.ml outcome.ml"
   * not-bsd
-  ** bytecode
+  ** windows
+  *** bytecode
```

Comment:
Isn't this specifying that `bytecode` should be executed _only_ on Windows? What am I missing?

Reply:
Gah - pushed the wrong version!

Reply:
Corrected version pushed 🙈

Reply:
Wait, doesn't the `native` item below also need incrementing? (Or do you intentionally want to keep running the native test because it is not flaky enough? I would disable the whole test.)

Reply:
Slightly weird, but fine with me.